code/datums/ai/basic_mobs/generic_controllers/hostile_controllers.dm

```diff
@@ -15,6 +15,14 @@
 		/datum/ai_planning_subtree/attack_obstacle_in_path/prowl,
 	)
 
+/datum/ai_controller/basic_controller/simple/simple_hostile_obstacles/ventcrawler
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/attack_obstacle_in_path,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree,
+		/datum/ai_planning_subtree/ventcrawl,
+	)
+
```

Comment:
Can you make sure the ventcrawling works properly since this is its first time being introduced into production? IIRC They won't ventcrawl unless you give them a specific goal to ventcrawl to (BB_VENTCRAWL_FINAL_TARGET)